src/google/protobuf/compiler/command_line_interface.h

```diff
@@ -244,7 +244,9 @@ class LIBPROTOC_EXPORT CommandLineInterface {
                             string* error);
 
   // Implements --encode and --decode.
-  bool EncodeOrDecode(const DescriptorPool* pool);
+  bool EncodeOrDecode(const DescriptorPool* pool,
```

Comment:
this changes the ABI ... i'm not sure what the protobuf policy is on this.  i'm guessing we should continue to support the old API too ?